src/GovUk.Education.ExploreEducationStatistics.Common/Utils/Html/HtmlToTextConverter.cs

```diff
@@ -145,9 +148,13 @@ private void AddSpacing(IElement element)
 
             if (element.IsBlockType() || next is IElement nextElement && nextElement.IsBlockType())
             {
-                // If last two characters are new line characters, then don't
-                // append any more spacing as this would be excessive.
-                if (_builder[^2] == '\n' && _builder[^1] == '\n')
+                // Get last 4 chars to check their line endings. Windows uses CRLF line endings (\r\n),
+                // so there may be up to 4 chars, instead of 2 for LF (\n) when using Linux / Mac.
+                var currentEnding = _builder.Substring(^4..);
+
+                // If already have two line endings, then don't
+                // append more spacing as this would be excessive.
+                if (currentEnding.EndsWith($"{Environment.NewLine}{Environment.NewLine}"))
```

Comment:
I've improved the spacing behaviour a bit so that it's more consistent to avoid appending too many lines as you've suggested.

As it's a bit smarter about how it appends lines, I've also applied it more generally in `ParseElement`. A nice improvement as lines don't need to be appended on a case-by-case basis anymore.